src/baustein.js

```diff
@@ -168,10 +168,16 @@ function noop () {
  * @param {String} str
  * @returns {String}
  */
-function toCamelCase (str) {
-    return str.replace(/\-(.)/g, function (a,b) {
-        return b.toUpperCase();
-    });
+function toCamelCase(str) {
+    str = str.split('-');
```

Comment:
might be slightly more readable to give the new variable a new name. does a new var vs re-use impact performance?